operations/app/src/modules/front_door/variables.tf

```diff
@@ -1,29 +1,29 @@
 variable "environment" {
-  type = string
+  type        = string
   description = "Target Environment"
 }
 
 variable "resource_group" {
-  type = string
+  type        = string
   description = "Resource Group Name"
 }
 
 variable "resource_prefix" {
-  type = string
+  type        = string
   description = "Resource Prefix"
 }
 
 variable "location" {
-  type = string
+  type        = string
   description = "Function App Location"
 }
 
 variable "https_cert_names" {
-  type = list
+  type        = list(any)
```

Comment:
Is this supposed to be `any`? Should it be `object` or `string` instead?

Reply:
Fixed